conf/scalardb-custom-values.yaml

```diff
@@ -71,4 +98,19 @@ scalardb:
                 operator: In
                 values:
                   - scalardbpool
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - scalardb-server
```

Comment:
```suggestion
                    - scalardb
```
Ditto.

Reply:
I have updated it as suggested.
Thank you.